api/v1/ptpconfig_types.go

```diff
@@ -61,12 +61,15 @@ type PtpConfigList struct {
 }
 
 type PtpProfile struct {
-	Name                  *string            `json:"name"`
-	Interface             *string            `json:"interface,omitempty"`
-	Ptp4lOpts             *string            `json:"ptp4lOpts,omitempty"`
-	Phc2sysOpts           *string            `json:"phc2sysOpts,omitempty"`
-	Ptp4lConf             *string            `json:"ptp4lConf,omitempty"`
-	PtpSchedulingPolicy   *string            `json:"ptpSchedulingPolicy,omitempty"`
+	Name        *string `json:"name"`
+	Interface   *string `json:"interface,omitempty"`
+	Ptp4lOpts   *string `json:"ptp4lOpts,omitempty"`
+	Phc2sysOpts *string `json:"phc2sysOpts,omitempty"`
+	Ptp4lConf   *string `json:"ptp4lConf,omitempty"`
+	// +kubebuilder:validation:Enum=SCHED_OTHER;SCHED_FIFO;
```

Comment:
I think `SCHED_OTHER` is not valid here as there is nothing in the linuxptp

Reply:
SCHED_OTHER will have the same behaviour as just not setting it.